virt/storage/virt-storage-config-overview.adoc

```diff
@@ -0,0 +1,69 @@
+
+The following storage configuration tasks are mandatory:
+
+xref:../../post_installation_configuration/storage-configuration.adoc#defining-storage-classes_post-install-storage-configuration[Configure a default storage class]::
+
+You must configure a default storage class for your cluster. Otherwise, the cluster cannot receive automated boot source updates.
+
+xref:../../virt/storage/virt-configuring-storage-profile.adoc#virt-configuring-storage-profile[Configure storage profiles]::
+
+You must configure storage profiles if your storage provider is not recognized by CDI. A storage profile provides recommended storage settings based on the associated storage class.
+
+The following storage configuration tasks are optional:
+
+xref:../../virt/storage/virt-reserving-pvc-space-fs-overhead.adoc#virt-reserving-pvc-space-fs-overhead[Reserve additional PVC space for file system overhead]::
+
+By default, 5.5% of a file system PVC is reserved for overhead, reducing the space available for VM disks by that amount. You can configure a different overhead value.
+
+xref:../../virt/storage/virt-configuring-local-storage-with-hpp.adoc#virt-configuring-local-storage-with-hpp[Configure local storage by using the hostpath provisioner]::
+
+You can configure local storage for virtual machines by using the hostpath provisioner (HPP). When you install the {VirtProductName} Operator, the HPP Operator is automatically installed.
+
+xref:../../virt/storage/virt-enabling-user-permissions-to-clone-datavolumes.adoc#virt-enabling-user-permissions-to-clone-datavolumes[Configure user permissions to clone data volumes between namespaces]::
+
+You can configure RBAC roles to enable users to clone data volumes between namespaces.
+
+[id="cdi-configuration-tasks"]
+== Containerized Data Importer
+
+You can perform the following Containerized Data Importer (CDI) configuration tasks:
+
+xref:../../virt/storage/virt-configuring-cdi-for-namespace-resourcequota.adoc#virt-configuring-cdi-for-namespace-resourcequota[Override the resource request limits of a namespace]::
+You can configure CDI to import, upload, and clone VM disks into namespaces that are subject to CPU and memory resource restrictions.
+
+xref:../../virt/storage/virt-preparing-cdi-scratch-space.adoc#virt-preparing-cdi-scratch-space[Configure CDI scratch space]::
+CDI requires scratch space (temporary storage) to complete some operations, such as importing and uploading VM images. During this process, CDI provisions a scratch space PVC equal to the size of the PVC backing the destination data volume (DV).
+
+[id="dv-configuration-tasks"]
+== Data volumes
+
+You can perform the following data volume configuration tasks:
+
+xref:../../virt/storage/virt-using-preallocation-for-datavolumes.adoc#virt-using-preallocation-for-datavolumes[Enable preallocation for data volumes]::
+
+CDI can preallocate disk space to improve write performance when creating data volumes. You can enable preallocation for specific data volumes.
```

Comment:
```suggestion
The CDI can preallocate disk space to improve write performance when creating data volumes. You can enable preallocation for specific data volumes.
```

Reply:
Outside of the Virt docs, we tend to use _pre-allocate_ over _preallocate_ at 24 instances to 7.  Maybe something to consider outside if this PR? (Preallocation is only used in the Virt docs.)